main.py

```diff
@@ -290,16 +291,17 @@ def make(self):
         fruitscore = 0
         score = 0
         
-        if os.path.exists("score.pkl")==False:
-            open('score.pkl','w+')
+        score_path = get_activity_root() + '/instance' + '/score.pkl'
+        if os.path.exists(score_path) is False:
```

Comment:
`if not os.path.exists(score_path)` would be better

Reply:
Fixed it @pro-panda